### ISSUE TITLE
Fix

### ISSUE THREAD

**Issue by allfreelancers** [MaksimRadziuk/Alfa#1]
- foter_payments-wrap
- popup menu - dropdown_menu -> li -> ul.dropdown_menu
- footer - caps tags (Li) and titles
- youmoney - yoomoney
- footer_soial-wrap
- img - images
- header and footer images - sprite (social)
- etc....

**Comment by allfreelancers** [comment]
@MaksimRadziuk fix please

**Comment by MaksimRadziuk** [comment]
@allfreelancers could you, please, describe with more details, what's wrong with popup menu
popup menu - dropdown_menu -> li -> ul.dropdown_menu ?

**Comment by allfreelancers** [comment]
@MaksimRadziuk semantic error of nested level li > ul

**Comment by MaksimRadziuk** [comment]
you are wrong https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

**Comment by allfreelancers** [comment]
@MaksimRadziuk [see](https://github.com/MaksimRadziuk/Alfa/blob/main/about.html#L145)

**Comment by MaksimRadziuk** [comment]
Sorry, I found the error

**Comment by MaksimRadziuk** [comment]
dropdown + "Li" fixed

**Comment by allfreelancers** [comment]
header and footer images - sprite (social)

**Comment by MaksimRadziuk** [comment]
it'll be fixed this night

**Comment by allfreelancers** [comment]
![image](https://user-images.githubusercontent.com/2994432/129187851-029cf22f-01f4-4e50-9abe-7e02b12cdc14.png)
all images must be in sprite

**Comment by allfreelancers** [comment]
@MaksimRadziuk https://prnt.sc/1nfnbmz ??

**Comment by MaksimRadziuk** [comment]
What does it mean "??"?

**Comment by allfreelancers** [comment]
`На главной https://maksimradziuk.github.io/Alfa/ проверяем https://prnt.sc/1nfnbmz`
Can you open link?

**Comment by MaksimRadziuk** [comment]
Hm... I think there is a misunderstanding. This plugin is an example of the dropdown select list. Customer asked to have possibility to enter letters to choose an appropriate option. This plugin has deviations from the design. So, I need customer's confirmation before implementing it everywhere.

**Comment by allfreelancers** [comment]
@MaksimRadziuk can you upload preview select list here? What about my list of fixes?

**Comment by MaksimRadziuk** [comment]
Yesterday I created sprites, because designer refused to do it. Now I'm working on them.
For select customization I used https://select2.org/. It's implemented in a first select box on the Main page https://maksimradziuk.github.io/Alfa/ to allow customer to check it and decide.

**Comment by allfreelancers** [comment]
Leave it as is and work on my correction. I will be rework all JS code on vuejs and there will still be discrepancies with the design

**Comment by MaksimRadziuk** [comment]
ok
Are you sure that there is a need in car make sprite here - https://prnt.sc/1nwtnj7
I'm wondering cause current solution will allow to add new make without any css update, but if we'll use sprite you'll need to do more work in future.

**Comment by allfreelancers** [comment]
Upload preview here plz. I can open there. Car marks you mean?

**Comment by MaksimRadziuk** [comment]
car make - company label (Audi, BMW etc)
ok, I'll notify you after completing

**Comment by allfreelancers** [comment]
do as much as possible

**Comment by MaksimRadziuk** [comment]
@allfreelancers updates completed and committed

**Comment by allfreelancers** [comment]
@MaksimRadziuk good job! Can you that too?
![image](https://user-images.githubusercontent.com/2994432/129519433-a3668f2d-9c0a-45f9-89bb-6cd2ce5e0ea0.png)
![image](https://user-images.githubusercontent.com/2994432/129520016-4c43539d-6b7d-41dc-a459-15ae024e5d07.png)
![image](https://user-images.githubusercontent.com/2994432/129519963-e1d5ce76-73cc-4e2e-bc53-fd1ee0166710.png)

Html looks much nicer!

**Comment by MaksimRadziuk** [comment]
Morning!
Yes, sure, I'll do it. But I'm extremely busy today. Is it ok to update the code tomorrow?

**Comment by allfreelancers** [comment]
@MaksimRadziuk Morning too! yes, of course, I wait for you long as you need! 
Good day for you!

**Comment by MaksimRadziuk** [comment]
@allfreelancers yesterday issues are fixed and committed

**Comment by allfreelancers** [comment]
@MaksimRadziuk Thank you! good job!

**Comment by allfreelancers** [comment]
@MaksimRadziuk Hi!
check images https://maksimradziuk.github.io/Alfa/contact.html 

<span class="img_wrap"><img src="img/icon-phone.png" alt=""></span> etc...

**Comment by MaksimRadziuk** [comment]
Hi @allfreelancers 
I'll fix it today

**Comment by MaksimRadziuk** [comment]
fixed and commited
